solvestack/settings.py

```diff
@@ -81,9 +81,13 @@
 # https://docs.djangoproject.com/en/3.0/ref/settings/#databases
 
 DATABASES = {
-    "default": {
-        "ENGINE": "django.db.backends.sqlite3",
-        "NAME": os.path.join(BASE_DIR, "db.sqlite3"),
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'solvestack_docker_1',
```

Comment:
Can you use environment variables here? That way it will inherit whatever's in our .env file 😄

Reply:
lol at one point I had it doing that!  will absolutely get that in there along with a matching .envexample file :)

Reply:
Excellent!